python/tank/authentication/user_impl.py

```diff
@@ -352,7 +352,10 @@ def to_dict(self):
         data = super(SessionUser, self).to_dict()
         data["login"] = self.get_login()
         data["session_token"] = self.get_session_token()
-        data["cookies"] = self.get_cookies()
+        # To preserve backward compatibility with older cores, we avoid
+        # serializing the cookies if therer are not any.
+        if self.get_cookies() is not None:
```

Comment:
I thought we already had that logic elsewhere. Is there a way to centralize it to avoid seeing the issue in another context?

Reply:
We do check for the presence/absence of cookies in the session_cache tests.

What exactly did you have in mind ?

Reply:
Spoke with JF, all is well. He was thinking of the session_cache tests.